documentation/jak1/gametasks.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 0
+description:
+---
+
+# Game Tasks
```

Comment:
not sure if we want a top-level `jak1` entry.  Maybe `documentation/game-reference/` would be better.  And then maybe game specific folders for knowledge that is unique to a particular game.